src/index.html

```diff
@@ -11,8 +11,51 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="use-credentials">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+
+      <div class="card__image"></div>
+
+      <div class="card__product-info">
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+      </div>
+
+      <div class="card__reviews-wrapper">
+        <div class="stars stars--4">
+          <span class="star"></span>
+          <span class="star"></span>
+          <span class="star"></span>
+          <span class="star"></span>
+          <span class="star"></span>
+        </div>
+
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__price-info">
+        <p class="card__text">Price:</p>
+        <p class="card__price">$2,199</p>
+      </div>
+
+      <button class="card__button"
+        aria-label="Buy Apple iMac"
+        data-qa="hover"
+      >
+        <div class="card__button-text">
+          BUY
+        </div>
+      </button>
+    </div>
```

Comment:
There is an extra closing `</div>` tag here. Please ensure that all opening tags have corresponding closing tags to prevent layout issues.